rkt/images.go

```diff
@@ -286,11 +282,16 @@ func (f *fetcher) fetchImageFrom(aciURL, ascURL, scheme string, ascFile *os.File
 	var key string
 	rem, ok, err := f.s.GetRemote(aciURL)
 	if err == nil {
+		stdout("rkt: found image from local store, skip fetching from %s", aciURL)
```

Comment:
if ok == true ?
Also ", skip fetching ... " seems a little bit too verbose to me ?